### PR TITLE
[release-4.5] Bug 1840253: Tighten cipher usage to library-go

### PR DESCRIPTION
**Pull request by jhadvig**
Manual backport of https://github.com/openshift/console/pull/6292

/assign @spadgett

### CODE REVIEW

**Review by spadgett — APPROVED** [review]
/lgtm

File: cmd/bridge/main.go [hunk 1]
```diff
@@ -591,9 +592,9 @@ func main() {
 		Handler: srv.HTTPHandler(),
 		// Disable HTTP/2, which breaks WebSockets.
 		TLSNextProto: make(map[string]func(*http.Server, *tls.Conn, http.Handler)),
-		TLSConfig: &tls.Config{
+		TLSConfig: oscrypto.SecureTLSConfig(&tls.Config{
 			CipherSuites: crypto.DefaultCiphers(),
```
Comment (spadgett) [inline cmd/bridge/main.go:596]: I think we can remove `crypto.DefaultCiphers()` now and associated module since we have library-go.

**Review by spadgett — CHANGES_REQUESTED** [review]
/lgtm cancel
/hold cancel

**Comment by jhadvig** [comment]
@spadgett comment addressed. Thanks for noticing!
PTAL

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**